sw/device/silicon_owner/bare_metal/BUILD

```diff
@@ -181,5 +181,5 @@ opentitan_functest(
     key = "multislot",
     ot_flash_binary = ":rom_ext_virtual_ottf_bl0_virtual",
     signed = True,
-    targets = ["cw310"],
+    targets = ["cw310_rom"],
```

Comment:
Same here, should we delete the `bitstream` definition on line 179?

Reply:
Done.